helper-cli/src/main/kotlin/commands/ExportPathExcludesCommand.kt

```diff
@@ -19,62 +19,57 @@
 import org.ossreviewtoolkit.utils.safeMkdirs
 
 import java.io.File
 
-@Parameters(
-    commandNames = ["export-path-excludes"],
-    commandDescription = "Export the path excludes to a path excludes file which maps repository URLs to the path " +
-            "excludes for the respective repository."
-)
-internal class ExportPathExcludesCommand : CommandWithHelp() {
-    @Parameter(
-        names = ["--path-excludes-file"],
-        required = true,
-        order = PARAMETER_ORDER_MANDATORY,
-        description = "The output path excludes file."
-    )
-    private lateinit var pathExcludesFile: File
+internal class ExportPathExcludesCommand : CliktCommand(
+    name = "export-path-excludes",
+    help = "Export the path excludes to a path excludes file which maps repository URLs to the path excludes for the " +
+            "respective repository."
+) {
+    private val pathExcludesFile by option(
+        "--path-excludes-file",
+        help = "The output path excludes file."
+    ).convert { it.expandTilde() }
+        .file(mustExist = false, canBeFile = true, canBeDir = false, mustBeWritable = false, mustBeReadable = false)
+        .required()
 
-    @Parameter(
-        names = ["--ort-result-file"],
-        required = true,
-        order = PARAMETER_ORDER_MANDATORY,
-        description = "The input ORT file from which the path excludes are to be read."
-    )
-    private lateinit var ortResultFile: File
+    private val ortResultFile by option(
+        "--ort-result-file",
+        help = "The input ORT file from which the path excludes are to be read."
+    ).convert { it.expandTilde() }
+        .file(mustExist = true, canBeFile = true, canBeDir = false, mustBeWritable = false, mustBeReadable = true)
+        .required()
 
-    @Parameter(
-        names = ["--repository-configuration-file"],
-        required = true,
-        order = PARAMETER_ORDER_MANDATORY,
-        description = "Override the repository configuration contained in the given input ORT file."
-    )
-    private lateinit var repositoryConfigurationFile: File
+    private val repositoryConfigurationFile by option(
+        "--repository-configuration-file",
+        help = "Override the repository configuration contained in the given input ORT file."
+    ).convert { it.expandTilde() }
+        .file(mustExist = true, canBeFile = true, canBeDir = false, mustBeWritable = false, mustBeReadable = true)
+        .required()
 
-    @Parameter(
-        names = ["--update-only-existing"],
-        required = false,
-        order = PARAMETER_ORDER_OPTIONAL,
-        description = "If enabled, only entries are exported for which an entry with the same pattern already exists."
-    )
-    private var updateOnlyExisting = false
+    private val updateOnlyExisting by option(
+        "--update-only-existing",
+        help = "If enabled, only entries are exported for which an entry with the same pattern already exists."
+    ).convert { it.expandTilde() }
```

Comment:
This should only be done for `File` options.

Reply:
Done.